docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+                    }
+                }
+            ],
+            "result": {
+                "name": "app",
+                "required": true,
+                "schema": {
+                    "$ref": "#/components/schemas/DeviceApp"
+                }
+            },
+            "errors": [
+                {
+                    "$ref": "#/components/errors/PermissionDenied"
+                },
+                {
+                    "$ref": "#/components/errors/UserCancel"
+                },
+                {
+                    "$ref": "#/components/errors/DeviceNotFound"
+                },
+                {
+                    "$ref": "#/components/errors/DeviceApplicationIsNotInstalled"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.device.bridge_app",
+            "summary": "Open a bridge between the thrid-party application and a given application to exchange APDU directly with only this app.",
+            "description": "Open a bridge between the thrid-party application and a given application to exchange APDU directly with only this app.",
+            "paramStructure": "by-name",
+            "tags": [
+                {
+                    "$ref": "#/components/tags/PermissionDeviceExchange"
+                }
+            ],
+            "params": [
+                {
+                    "name": "appName",
+                    "summary": "The name of the application to request on the device",
+                    "description": "The name of the application to request on the device",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ],
+            "result": {
+                "name": "deviceBridge",
+                "schema": {
+                    "$ref": "#/components/schemas/DeviceBridgeId"
+                }
+            },
+            "errors": [
+                {
+                    "$ref": "#/components/errors/PermissionDenied"
+                },
+                {
+                    "$ref": "#/components/errors/UserCancel"
+                },
+                {
+                    "$ref": "#/components/errors/DeviceNotFound"
+                },
+                {
+                    "$ref": "#/components/errors/DeviceApplicationIsNotInstalled"
+                }
+            ]
+        },
+        {
+            "name": "ledgerlive.device.bridge_dashboard",
+            "summary": "Open a bridge between the thrid-party application and the device dashboard to exchange APDU directly with the dashboard.",
+            "description": "Open a bridge between the thrid-party application and the device dashboard to exchange APDU directly with the dashboard.",
+            "paramStructure": "by-name",
+            "tags": [
+                {
+                    "$ref": "#/components/tags/PermissionDeviceExchange"
```

Comment:
Might be a specific permission for accessing the dashboard?

Reply:
Indeed,